### DISCUSSION TITLE
Using DynamoDB for Session Storage

### DISCUSSION THREAD

**Discussion by yuppe-kyupeen** [remix-run/remix#9458] [Q&A]
Hello,

I frequently use Remix and am continually impressed by its simple and elegant design. Thank you for creating such a great framework.

Currently, in my project, I am considering using DynamoDB as the session storage for a server-side rendered application. I noticed in the official Remix documentation (https://remix.run/docs/en/1.19.3/utils/sessions#createarctablesessionstorage-architect-amazon-dynamodb) that there is an API called createArcTableSessionStorage(). However, the documentation lacks specific guidance on how to connect to the endpoint, and I haven't found many examples through my searches.

Does anyone know of any projects or resources that could serve as a useful reference for this?

Thank you!

**Comment by kiliman** [comment]
You should be able to create your own `createDynamoSessionStorage`

Here's the Arc example. 

https://github.com/remix-run/remix/blob/main/packages/remix-architect/sessions/arcTableSessionStorage.ts

**Reply by kiliman** [reply]
I'm not familiar with arc but it looks like you configure arc tables separately. The session store will use the arc global object to access the table. See `getTable()` in the source file. 

As for dynamodb it looks like you do something similar.
